easybuild/easyconfigs/h/HISAT2/HISAT2-2.1.0_intel-fixes.patch

```diff
@@ -0,0 +1,13 @@
+--- hisat2-2.1.0/processor_support.h.orig	2017-10-05 16:50:19.290831227 +0200
```

Comment:
@smoors Please include a brief description of the patch + mention author or include reference to upstream source of patch

Reply:
sorry for not including the author (you) :)